lib/src/Uncertainty/Algorithm/MetaModel/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ ot_install_header_file (OTMetaModel.hxx)
 add_subdirectory (FunctionalChaos)
 add_subdirectory (Kriging)
 add_subdirectory (TensorApproximation)
+add_subdirectory (LinearModelAlgorithm)
```

Comment:
the folder can be named just LinearModel, but that's no a big deal

Reply:
I will rename it